From 86fb04aa58cd5355c63e8b962370446e9c34f04c Mon Sep 17 00:00:00 2001
From: Julie Jeongeun Kim <jkim@igalia.com>
Date: Tue, 16 Oct 2018 16:54:03 +0900
Subject: [PATCH] Update user directory

It updates user directory path for flounder from
"/home/root/" to "/home/0/"
---
 files/launch/WebAppMgr.env     | 2 +-
 files/launch/WebAppMgr.service | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/files/launch/WebAppMgr.env b/files/launch/WebAppMgr.env
index 1c0bc13..8d1a28e 100644
--- a/files/launch/WebAppMgr.env
+++ b/files/launch/WebAppMgr.env
@@ -26,7 +26,7 @@ WAM_SUSPEND_DELAY_IN_MS=250
 #fi
 
 # Set user data directory for WebAppMgr
-WAM_DATA_PATH="/home/root/wamdata"
+WAM_DATA_PATH="/home/0/wamdata"
 
 # ensure that wam data directories exist
 #mkdir -p ${WAM_DATA_PATH}
diff --git a/files/launch/WebAppMgr.service b/files/launch/WebAppMgr.service
index 42dcb3b..5a5fe0a 100644
--- a/files/launch/WebAppMgr.service
+++ b/files/launch/WebAppMgr.service
@@ -19,7 +19,7 @@ Wants=weston.service
 Type=simple
 OOMScoreAdjust=-1000
 EnvironmentFile=-/etc/default/WebAppMgr.env
-ExecStart=/usr/bin/WebAppMgr --no-sandbox --remote-debugging-port=9998 --user-data-dir="/home/root/wamdata" --webos-wam
+ExecStart=/usr/bin/WebAppMgr --no-sandbox --remote-debugging-port=9998 --user-data-dir="/home/0/wamdata" --webos-wam
 Restart=on-failure
 
 [Install]
-- 
2.14.1

